src/app/legacy/containers/PodcastPromo/Inline.jsx

```diff
@@ -33,35 +39,31 @@ const GEL_GROUP_1_WIDTH_360PX = '22.5rem';
 
 const ResponsivePodcastPromoWrapper = styled.div`
   ${({ dir }) => (dir === 'ltr' ? 'float: right;' : 'float: left;')}
-  background: ${props => props.theme.palette.LUNAR};
-  margin: ${GEL_SPACING_TRPL} 0;
+
+  margin: 0rem ${GEL_SPACING} ${GEL_SPACING_TRPL} ${GEL_SPACING};
   height: auto;
 
   @media (min-width: ${GEL_GROUP_1_SCREEN_WIDTH_MIN}) {
     width: 45%;
-    margin: ${GEL_SPACING_TRPL} ${GEL_SPACING_HLF};
+    margin: ${GEL_SPACING} ${GEL_SPACING} ${GEL_SPACING_TRPL} ${GEL_SPACING};
```

Comment:
Same as above, I think this can be: ${GEL_SPACING} ${GEL_SPACING} ${GEL_SPACING_TRPL}